index.html

```diff
@@ -12,7 +12,7 @@
       <div class="welcome-logo">Welcome</div>
       <nav class="nav">
         <button class="nav-toggle" id="btn" aria-label="toggle navigation">
-          <span class="hamburger"></span>
+          <li class="hamburger"></li>
```

Comment:
`li` tag is supposed to be inside an `ul` or `ol` tag only. Kindly remove it here as it has no semantic meaning.